app/helpers/contributor-concat-name.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export function contributorConcatName([givenName, familyName, fullName]) {
```

Comment:
This probably makes more sense as a computed property of user. Perhaps just call it `name`? Then you can just say `{{author.users.name}}` in the template and the logic is in the user model.